types/foundry-vtt-api/types/framework/entities/rolltable.d.ts

```diff
@@ -0,0 +1,86 @@
+declare class RollTables extends Collection<RollTable> {
+	entities: RollTable[];
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/foundry-vtt-api/types/framework/entities/rolltable.d.ts;
  Line: 2;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.